pom.xml

```diff
@@ -127,7 +129,16 @@
             <artifactId>aspectjtools</artifactId>
             <groupId>org.aspectj</groupId>
         </dependency>
-
+        <dependency>
+            <groupId>uk.gov.companieshouse</groupId>
+            <artifactId>private-api-sdk-java</artifactId>
```

Comment:
`api-sdk-manager-java-library` brings in `private-api-sdk-java` as a dependency.

Reply:
It does, however the latest java 11 compatible version (1.0.9 of this library is using private-sdk-java v2.0.265) which is very out of date. By directly stating a version and installing that, we can get more granular control over the sdk version.

Reply:
That could mean you would end up with two different versions of `private-api-sdk-java` in your dependencies? I have assumed it is cleaner to update the `api-sdk-manager-java-library` to a version that brings in the version of `private-api-sdk-java` that you want.

Reply:
If you declare a dependency in your project, Maven will use this version even if another library declares another version. This is provable if you download the project and inspect the external libraries.

Reply:
Good point. 👍